inference-engine/src/vpu/graph_transformer/src/parsed_config.cpp

```diff
@@ -146,58 +149,59 @@ void ParsedConfig::parse(const std::map<std::string, std::string>& config) {
+    setOption(_compileConfig.hwWhiteList,                              config, ie::MYRIAD_HW_WHITE_LIST, parseStringSet);
+    setOption(_compileConfig.hwBlackList,                              config, ie::MYRIAD_HW_BLACK_LIST, parseStringSet);
 
     // Priority is set to VPU configuration file over plug-in config.
-    setOption(_compileConfig.customLayers, config, VPU_CONFIG_KEY(CUSTOM_LAYERS));
+    setOption(_compileConfig.customLayers,                             config, ie::MYRIAD_CUSTOM_LAYERS);
     if (_compileConfig.customLayers.empty()) {
-        setOption(_compileConfig.customLayers, config, CONFIG_KEY(CONFIG_FILE));
+        setOption(_compileConfig.customLayers,                         config, CONFIG_KEY(CONFIG_FILE));
     }
 
-    setOption(_compileConfig.numSHAVEs, config, VPU_CONFIG_KEY(NUMBER_OF_SHAVES), parseInt);
-    setOption(_compileConfig.numCMXSlices, config, VPU_CONFIG_KEY(NUMBER_OF_CMX_SLICES), parseInt);
-    setOption(_compileConfig.numExecutors, config, VPU_MYRIAD_CONFIG_KEY(THROUGHPUT_STREAMS), parseInt);
+    setOption(_compileConfig.numSHAVEs,                                config, ie::MYRIAD_NUMBER_OF_SHAVES, parseInt);
+    setOption(_compileConfig.numCMXSlices,                             config, ie::MYRIAD_NUMBER_OF_CMX_SLICES, parseInt);
+    setOption(_compileConfig.numExecutors,                             config, ie::MYRIAD_THROUGHPUT_STREAMS, parseInt);
 
     if ((_compileConfig.numSHAVEs < 0 && _compileConfig.numCMXSlices >= 0) ||
         (_compileConfig.numSHAVEs >= 0 && _compileConfig.numCMXSlices < 0)) {
         THROW_IE_EXCEPTION << "You should set both option for resource management: VPU_NUMBER_OF_CMX_SLICES and VPU_NUMBER_OF_SHAVES";
     }
 
-    setOption(_compileConfig.ioStrides, config, VPU_CONFIG_KEY(TENSOR_STRIDES), parseStrides);
+    setOption(_compileConfig.ioStrides,                                config, ie::MYRIAD_TENSOR_STRIDES, parseStrides);
 
-    setOption(_printReceiveTensorTime, switches,    config, VPU_CONFIG_KEY(PRINT_RECEIVE_TENSOR_TIME));
-    setOption(_perfCount,              switches,    config, CONFIG_KEY(PERF_COUNT));
-    setOption(_perfReport,             perfReports, config, VPU_CONFIG_KEY(PERF_REPORT_MODE));
+    setOption(_printReceiveTensorTime,                       switches, config, ie::MYRIAD_ENABLE_RECEIVING_TENSOR_TIME);
+    setOption(_perfCount,                                    switches, config, CONFIG_KEY(PERF_COUNT));
+    setOption(_perfReport,                                perfReports, config, ie::MYRIAD_PERF_REPORT_MODE);
 
 IE_SUPPRESS_DEPRECATED_START
-    setOption(_compileConfig.inputScale, config, VPU_CONFIG_KEY(INPUT_NORM), parseFloatReverse);
-    setOption(_compileConfig.inputBias, config, VPU_CONFIG_KEY(INPUT_BIAS), parseFloat);
+    setOption(_compileConfig.hwOptimization,                 switches, config, VPU_CONFIG_KEY(HW_STAGES_OPTIMIZATION));
```

Comment:
Why something as variaale, some as `CONFIG_KEY` macro and some as `VPU_CONFIG_KEY`?

Reply:
`VPU_CONFIG_KEY` is left as deprecated usage
`CONFIG_KEY` is common api
The main reason to still use macros is long detailed names `InferenceEngine::PluginConfigParams::KEY_...` or `InferenceEngine::VPUConfigParams::KEY_VPU_MYRIAD...` respectively. Anyway, I have nothing against replacing them with variables